### ISSUE TITLE
Nonstandard Implementation

### ISSUE THREAD

**Issue by lovelyczli** [tobyperrett/trx#9]
In the few-shot learning setting,  a dataset is split into training, validation, and testing splits.  However, all models are trained on training splits, and the validation set is used for validation, and the test set is ONLY used for computing the final result (because we do not know the specific distribution on the test set).
Common evaluation: Select the best model (higher accuracy) on the val set when training, and employ this model on the test set (only once). And some works select the N episodes model, but the hyperparameter N is obtained by val set.

However, your implementation does NOT use the val set, and meanwhile, the test set is used repeatedly. 

And it is unfair to compare to previous works.

**Comment by tobyperrett** [comment]
Our results are obtained the correct way, and hyperparams etc. are chosen using the validation set. Please see this answer on how to use the validation set with this codebase: https://github.com/tobyperrett/trx/issues/3

This is also explained in the paper in Section 4.1.
